src/main/java/com/kodilla/ecommercee/CartController/CartDomainDto/CartItemDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.CartController.CartDomainDto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class CartItemDto {
+
+    private Long id;
+    private String name;
```

Comment:
I' might be shooting blanks here but maybe we need list with product id List<Long>productId insted of Product name, description, price and group (you have all this information in Product table). I assum, after doing GetMapping you want to get cartId and list of products. Although specs doesn't look like this,  but in our specs you have cart with only one product and I wonder how it should look like when we have more then one product in one cart. I don't know. You might want to check it with our PM (Andrzej)

Reply:
This is CartItemDto, not CartDto. List<CartItemDto> is in CartDto.